mempool/v1/mempool.go

```diff
@@ -401,22 +400,25 @@ func (txmp *TxMempool) Update(
 	}
 
 	for i, tx := range blockTxs {
+		// The tx hash corresponding to the originating transaction. Set to the
+		// current tx hash initially, but can can change in case of a malleated tx.
```

Comment:
```suggestion
		// current tx hash initially, but can change in case of a malleated tx.
```

Reply:
Resolved in [7eeef3f](https://github.com/celestiaorg/celestia-core/pull/834/commits/7eeef3f4156b5ef6786e36fdad89588aadef3b45)